build/tmp/spotless-register-dependencies

```diff
@@ -0,0 +1 @@
+1
```

Comment:
should this file be here? or should we add it to the git ignore?

Reply:
after some local testing, it appears this file is generated on build and can be removed safely.